### PR TITLE
added grey color to buyMeACoffeButton and ko-fiButton

### PR DESCRIPTION
**Pull request by maxmitchenko**
added grey color to buyMeACoffeButton and ko-fiButton to show visually that it is disabled

### CODE REVIEW

**Comment by Flajt** [comment]
Just skimmed it, seems good. I'm thinking if it's worth to add an IgnorePointer or similar to prevent User input aswell if grey is selected as color?

**Comment by maxmitchenko** [comment]
> Just skimmed it, seems good. I'm thinking if it's worth to add an IgnorePointer or similar to prevent User input aswell if grey is selected as color?

Please, take a look on a new commit with isEnabled value

File: lib/donationButtons/ko-fiButton.dart [hunk 1]
```diff
@@ -3,7 +3,7 @@ import 'package:simple_icons/simple_icons.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
 ///The 4 Official Kofi Button Colors
-enum KofiColor { Blue, Black, Orange, Red }
+enum KofiColor { Blue, Black, Orange, Red, Grey }
```
Comment (Flajt) [inline lib/donationButtons/ko-fiButton.dart:6]: If you implement the Grey color here as well, I think we should also implement the input blocking?

Comment (maxmitchenko) [reply lib/donationButtons/ko-fiButton.dart:6]: Yes, and I have implemented it in second commit -  148fc20b6239111cf45d3291d5961a14c4fae5c5
Or what do you mean?

Comment (Flajt) [reply lib/donationButtons/ko-fiButton.dart:6]: Maybe I'm missing it but for the `ko-fiButton.dart`, there is non implemented, only for `buyMeACoffeButton.dart`, if I see it correctly.
EDIT: There is the grey color option, but not the `isEnabled` check.

Comment (maxmitchenko) [reply lib/donationButtons/ko-fiButton.dart:6]: Ou, yes, I did not add it. Added in my last commit, please check.

**Review by Flajt — APPROVED** [review]
Sounds good, thanks a lot.

And thanks a lot for contributing, its nice to see.

**Pull request merged**